src/main/java/zuve/searchablechests/ChestEventHandler.java

```diff
@@ -117,7 +121,7 @@ public void onOpenGui(GuiOpenEvent event) {
 	public void onBackground(GuiContainerEvent.DrawBackground event) {
 		if (searchField != null) {
 			mc.getTextureManager().bindTexture(searchBar);
-			AbstractGui.blit(event.getGuiContainer().getGuiLeft() + 80, event.getGuiContainer().getGuiTop() + 4, 0.0F,
+			AbstractGui.blit(ms, event.getGuiContainer().getGuiLeft() + 79, event.getGuiContainer().getGuiTop() + 4, 0.0F,
```

Comment:
This can instead use `event.getMatrixStack()`, and we can remove the `ms` variable altogether.